src/Tools/dotnet-trace/CommandLine/Commands/ListProfilesCommandHandler.cs

```diff
@@ -104,6 +148,44 @@ public static Command ListProfilesCommand()
                 "Captures ADO.NET and Entity Framework database commands")
         };
 
+        internal static IEnumerable<Profile> LinuxPerfEventProfiles { get; } = new[] {
+            new Profile(
+                "kernel-cpu",
+                providers: Array.Empty<EventPipeProvider>(),
+                description: "Kernel CPU sampling (perf-based), emitted as Universal.Events/cpu, for precise on-CPU attribution."),
+            new Profile(
+                "kernel-cswitch",
```

Comment:
Can I suggest we name these:
kernel-cpu -> cpu-samping
kernel-cswitch -> thread-time

I'm making an educated guess users will relate to those names a little better.

Reply:
Done, thanks!